### ISSUE TITLE
Bluetooth service missing feature

### ISSUE THREAD

**Issue by Gmrdrm** [libretro/Lakka-LibreELEC#1136]
Hello 

despite being possible to configure bluetooth controllers in the latest nightly build by plugging in devices at first via usb and then unplugging them, it is not possible to use the devices with bluetooth once this is done the first time and it is not possible to forget the devices in order to attempt another pairing test

thanks for support

**Comment by Ntemis** [comment]
This is a point/bug that needs to be reported to the retroarch repo and not here.
The retroarch menu GUI must handle this and this is what lakka use.
Please open a new issue here https://github.com/libretro/RetroArch reporting the issue.
